src/lib/server/index.js

```diff
@@ -0,0 +1,31 @@
+import express from "express";
+import logger from "../logger";
+import { filename } from "../logsColorScheme";
+import requestHandler from "./requestHandler";
+
+const isProduction = process.env.NODE_ENV === "production";
+//@TODO: read host from config file
+const host = "localhost";
+//@TODO: understand why different port numbers between prod and dev
```

Comment:
In development mode, we have webpack-dev-tools serving assets on one port and express serving the server side rendering on another port. In development mode, we proxy requests to the webpack dev tools server so that you don't have to point your code to two different ports. In production we are not proxying to another server, but instead serving the static assets ourselves.

https://github.com/TrueCar/gluestick/blob/develop/src/commands/start-client.js#L114

If I remember correctly, a request comes into the express app using the webpack-dev-server. If it finds the asset, then it returns the asset, if not then if tries to pass it along to the server side rendering port. So in development mode, you access the app on the webpack dev server route.

In production, we actually mount /assets to point to the pre-bundled assets.

I'm fairly certain this can be done better.